### DISCUSSION TITLE
请问partitions加载index时是否回加载完整的collection的index

### DISCUSSION THREAD

**Discussion by kkpssr** [milvus-io/milvus#30831] [Q&A and General discussion]
您好，请问collection中加载特定分区，分区的索引是独立的还是整个collection的一部分，加载特定分区时是否要加载collection的完整索引。

**Comment by yhmo** [comment]
每个collection有N个Partition
每个Partition有N个Segment
每个Segment有N行数据，列式存储，每个字段的数据存一列，其中向量列的数据建索引，该索引只服务于该Segment，独立于其他Segment的索引
加载某个Partition，就是加载它的N个Segment里的索引，没有索引的就加载原始向量数据

关键点： 数据是分片(segment)管理，每个分片都是独立存储，独立索引，独立加载，独立计算。

**Reply by kkpssr** [reply]
了解了，谢谢
